packages/test-utilities/lib/create-test-client.ts

```diff
@@ -25,6 +26,8 @@ const defaultOptions = () => ({
   spanAttributesSource,
   schema,
   plugins: () => [],
+  appState: 'starting',
+  setAppState: () => AppState,
```

Comment:
```suggestion
  setAppState: (appState: AppState) => void,
```

Reply:
In this case `'appState' is declared but its value is never read`
Maybe:  `setAppState: (appState: AppState) => {appState} `?